src/quicklist.c

```diff
@@ -1138,7 +1138,7 @@ int quicklistNext(quicklistIter *iter, quicklistEntry *entry) {
     entry->node = iter->current;
 
     if (!iter->current) {
-        D("Returning because current node is NULL")
+        D("Returning because current node is NULL");
         return 0;
```

Comment:
Interesting. Without the semicolon works in the normal case, but if REDIS_TEST_VERBOSE is defined, D expands to a `do { ... } while(0)` and it wouldn't compile here, so the verbose testing was broken (until now).

To make sure D without semicolon isn't added again, we can define D as something which needs to be followed by a semicolon, such as `do {} while(0)`. Just an idea.